### ISSUE TITLE
"Right-click on a .prompty file to quickly generate integration snippets" is not working.

### ISSUE THREAD

**Issue by 55yoko** [microsoft/prompty#12]
I right-clicked on the prompty file in Vscode and selected "Add prompt flow Code", but the file was not created. The Semantic kernel and Langchain were also not created.
![This is screen capture.](https://github.com/microsoft/prompty/assets/141200555/36a23a9b-c7e2-45be-9fd1-54a8a26a451b)

Visual Studio Code:Version: 1.89.1 
OS:mac Sonoma 14.5（23F79）
Prompty: v0.1.2024053008

**Comment by sethjuarez** [comment]
Uh oh! We will take a look. Thanks for the flag.

**Comment by wayliums** [comment]
@55yoko  can you share the prompty file here?

**Comment by 55yoko** [comment]
@wayliums 

This is a file that I just made this from right click and select "New prompty" , then I changed only azure_endpoint and azure_deployment. 

`---
name: ExamplePrompt
description: A prompt that uses context to ground an incoming question
authors:
  - Seth Juarez
model:
  api: chat
  configuration:
    type: azure_openai
    azure_endpoint: https://************.openai.azure.com
    azure_deployment: gpt-35-turbo
  parameters:
    max_tokens: 3000
sample:
  firstName: Cassie
  context: >
    The Alpine Explorer Tent boasts a detachable divider for privacy, 
    numerous mesh windows and adjustable vents for ventilation, and 
    a waterproof design. It even has a built-in gear loft for storing 
    your outdoor essentials. In short, it's a blend of privacy, comfort, 
    and convenience, making it your second home in the heart of nature!
  question: What can you tell me about your tents? Give me more information about the features?
---

system:
You are an AI assistant who helps people find information. As the assistant, 
you answer questions briefly, succinctly, and in a personable manner using 
markdown and even add some personal flair with appropriate emojis.

# Customer
You are helping {{firstName}} to find answers to their questions.
Use their name to address them in your responses.

# Context
Use the following context to provide a more personalized response to {{firstName}}:
{{context}}

user:
{{question}}
`

**Comment by wayliums** [comment]
@55yoko can you enable dev tools in VSCode, and see if there's any errors?
![image](https://github.com/microsoft/prompty/assets/11469098/db17eed4-f22f-4756-844f-793589c4328a)

And meanwhile, to unblock you, this is the PF code generated

```
import json

from pathlib import Path
folder = Path(__file__).parent.absolute().as_posix()

from promptflow.core import tool, Prompty

@tool
def flow_entry(    
      firstName: any,
      context: any,
      question: any
) -> str:
  # path to prompty (requires absolute path for deployment)
  path_to_prompty = folder + "/basic.prompty"

  # load prompty as a flow
  flow = Prompty.load(path_to_prompty)
 
  # execute the flow as function
  result = flow(
    firstName = firstName,
    context = context,
    question = question
  )

  return result

if __name__ == "__main__":
   json_input = '''{
  "firstName": "Cassie",
  "context": "The Alpine Explorer Tent boasts a detachable divider for privacy,  numerous mesh windows and adjustable vents for ventilation, and  a waterproof design. It even has a built-in gear loft for storing  your outdoor essentials. In short, it's a blend of privacy, comfort,  and convenience, making it your second home in the heart of nature!\\n",
  "question": "What can you tell me about your tents? Give me more information about the features?"
}'''
   args = json.loads(json_input)

   result = flow_entry(**args)
   print(result)
```

**Comment by 55yoko** [comment]
@wayliums 
Should I look at the console log? When I opened it, the following log was displayed. Even if I right-clicked to add code, the console log did not increase.
![Uploading consolelog.png…]()

**Comment by wayliums** [comment]
@55yoko  thanks for sharing. I can't repro the issue. Let me try adding a bit more logging first.

**Comment by wayliums** [comment]
@55yoko  i hope the sample I provided above would unblock you, right?

**Comment by sethjuarez** [comment]
Will close since the issue has been open for a bit. Feel free to submit a new issue if it's not fixed.

**Comment by btcpbordeaux** [comment]
@sethjuarez this is not fixed. Same issue, switched to preview and now the new option for Add Prompty Code does generate .env and prompty.json files. Unfortunately no other integration snippets are generated. No logs are generated from prompty either for the langchain, semantic kernel, or prompt flow snippet generators. Shall I submit a new issue?

**Comment by sethjuarez** [comment]
Yeah - can you submit a new issue - sorry about that! Let's see if we can't sort it out.

**Comment by sethjuarez** [comment]
Closing this - found some bugs and made some fixes to the codegen.
